sql/moz-fx-data-shared-prod/search_derived/acer_cohort_v1/init.sql

```diff
@@ -0,0 +1,9 @@
+-- Query for search_derived.acer_cohort_v1
+-- Pulls all client_ids that were using an Acer distribution in 2017-2022
+SELECT DISTINCT
+  client_id
+FROM
+  `moz-fx-data-shared-prod.telemetry.clients_daily`
+WHERE
+  submission_date >= '2017-01-08'
```

Comment:
```suggestion
  submission_date >= '2017-01-08'
  AND submission_date <= '2023-01-31'
```
I'd add an upper bound to make this init query idempotent.